src/desdemona/tasks/kafka.clj

```diff
@@ -2,6 +2,12 @@
   (:require [taoensso.timbre :refer [info]]
             [cheshire.core :as json]))
 
+(defn deserialize-message-json [bytes]
+  (try
+    (json/parse-string (String. bytes "UTF-8"))
```

Comment:
premature optimization is the root of all evil, but you can make this zero-copy by using [ztellman's byte-streams][bs], specifically by converting to a reader by using `bs/to-reader`, which means that you can use `json/parse-stream`. I probably wouldn't mention that if this wasn't on the critical path.

More importantly though, this is where we can convert the keys to keywords. You add the super obvious and intuitive and well-documented third argument `true` and HEY PRESTO strings. However, that means that you end up with keywords like `:_parsed`, `:ALLCAPS` and `:LOUD_SHOUTING` which don't really look like the kebab-case things you want. That bothers me (I'm willing to look past that), but if it bothers you too, I've written some code that solves that for you. Of course, that code lives in the [utils namespace of a browser-side Reagent library][lulz]...

[bs]: https://github.com/ztellman/byte-streams
[lulz]: https://github.com/RackSec/wilson/blob/master/src/wilson/utils.cljc#L25